app/views/configuration/_ui_2.html.haml

```diff
@@ -58,6 +59,19 @@
                   = ui_lookup(:tables => table_view_name)
                 .col-md-8
                   %ul.list-inline= render_view_buttons(resource, @edit[:new][:views][resource])
+          %fieldset
+            %h3
+              = _('VM Visibility')
+            .form-group
+              %label.col-md-8.control-label
+                = _("Set visibility of VMs in trees. Warning: Showing VMs may cause performance issues. So better set it to no.")
```

Comment:
@ZitaNemeckova how about move the note to the next line below checkbox, how about "Note: Turning this option on can cause performance issues"

And maybe label for checkbox can be changed to "Show VMs in Explorer tree"